src/subdomains/staking/staking.module.ts

```diff
@@ -31,6 +32,7 @@ import { StakingKycCheckService } from './infrastructure/staking-kyc-check.servi
     SharedModule,
     UserModule,
     PayInModule,
+    TransactionModule,
```

Comment:
TransactionModule is part of IntegrationModule. Should we rather import Integration here?

Reply:
fixed in #42